plugins/Wallet/js/components/newwalletdialog.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+
+const NewWalletDialog = ({visible, password, seed, actions}) => {
+	if (!visible) {
+		return (
+			<div></div>
```

Comment:
I agree.  Or at least, it should be consistent.  I will change the pattern to consistently be the parent deciding which elements to render, as is done in `lockscreen.js`